makejdk.sh

```diff
@@ -219,33 +218,33 @@ buildAndTestOpenJDKViaDocker()
   # Keep is undefined so we'll kill the docker image
 
   if [[ "$KEEP" == "true" ]] ; then
-     if [ $(docker ps -a | grep openjdk_container | wc -l) == 0 ]; then
+     if [ "$(docker ps -a | grep -c openjdk_container)" == 0 ]; then
          echo "${info}No docker container found so creating one${normal}"
          docker build -t $CONTAINER docker/jdk8u/x86_64/ubuntu
      fi
   else
      echo "${info}Building as you've not specified -k or --keep"
-     echo $good
+     echo "$good"
      docker ps -a | awk '{ print $1,$2 }' | grep $CONTAINER | awk '{print $1 }' | xargs -I {} docker rm -f {}
      docker build -t $CONTAINER docker/jdk8u/x86_64/ubuntu
-     echo $normal
+     echo "$normal"
   fi
 
-  docker run --privileged -t -v $WORKING_DIR/$OPENJDK_REPO_NAME:/openjdk/jdk8u/openjdk --entrypoint build.sh $CONTAINER
+  docker run --privileged -t -v "$WORKING_DIR/$OPENJDK_REPO_NAME:/openjdk/jdk8u/openjdk" --entrypoint build.sh "$CONTAINER"
 
   testOpenJDKViaDocker
 
   CONTAINER_ID=$(docker ps -a | awk '{ print $1,$2 }' | grep openjdk_container | awk '{print $1 }'| head -1)
 
   if [[ "${COPY_TO_HOST}" == "true" ]] ; then
-    echo "Copying to the host with docker cp $id:/openjdk/jdk8u/OpenJDK.tar.gz $TARGET_DIR"
-    docker cp $CONTAINER_ID:/openjdk/jdk8u/OpenJDK.tar.gz $TARGET_DIR
+    echo "Copying to the host with docker cp $CONTAINER_ID:/openjdk/jdk8u/OpenJDK.tar.gz $TARGET_DIR"
```

Comment:
I would say filter all discussions and then apply it on top of #54 and use that as our final guidelines.